docs/get-started/icon-libraries.njk

```diff
@@ -0,0 +1,153 @@
+---
+layout: layout-basic.njk
+title: Icon libraries
+order: 3
+tags:
+  - getstarted
+---
+
+<!-- Body -->
+
+{% import 'component/get-started.njk' as getstarted %}
+
+{% call getstarted.section("Introduction") -%}
+
+  <p>Icons are small visual representations that help us depict a variety of ideas and technologies. Our icons are open and straightforward, just like our brand. To see our repo of icons, visit <a href="https://docs.google.com/presentation/d/1SRhy8-bYBgaA3Jsi1t_Fxz-Yo9ORgdRy5Kec9hg_wSM/edit#slide=id.gb24fcfdda7_14_0">this page</a>.</p>
+
+{%- endcall %}
+
+{% call getstarted.section("Using libraries") -%}
+
+  <p>Our libraries require access to Adobe Creative Cloud through a Red Hat account. These libraries are intended for designers who regularly use Adobe CC apps as part of their role at Red Hat. Access will not be granted to accounts that are not part of the Red Hat domain.</p>
+
+  <rh-cta variant="secondary">
+    <a href="https://url.corp.redhat.com/adobecc">Request a Creative Cloud license</a>
+  </rh-cta>
+
+{%- endcall %}
+
+{% call getstarted.section("Request access") -%}
+
+  <p>To request access to the Adobe CC libraries containing brand assets, fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSeZbyEh0CGJrujIocHuR4ZMWh_uew-IeUNWewBOb3GBnUHTrA/viewform?usp=sf_link">this form</a>. Please allow a week or less to receive access. Once we accept your request, you will receive invites to join the <strong>three libraries</strong> below. These libraries will then sync to your Adobe CC design programs and you can view the assets in the <em>Libraries</em> panel.</p>
```

Comment:
The titles of the three libraries should be listed out as text, instead of just being shown as an image.